docs/status.md

```diff
@@ -18,128 +18,128 @@ one of the following tracking labels.
 
  - Generic labels
     - **yes**: complete
-    - **wip**: semi-complete: Work in progress or under review.
     - **no**: not complete yet, but part of [the roadmap](https://github.com/openxla/stablehlo#roadmap).
- - Customized labels
-    - Verifier
-       - **match-xla**:  verifier in sync with  [XLA semantics](https://www.tensorflow.org/xla/operation_semantics).
-       - **match-spec**: verifier in sync with [StableHLO semantics](https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md).
-
+ - Customized labels for Verifier and Type Inference 
+    - **yes\***:  in sync with  [XLA semantics](https://www.tensorflow.org/xla/operation_semantics).
+    - **yes**: in sync with [StableHLO semantics](https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md).
+    - **yes(need-revisit)**: implemented but need revisit for the sync with XLA or spec
+    - **infeasible**: infeasible to implement by design
+    
 ## Status
 
-| StableHLO Op (114) | Specification (21) | Verification | Type Inference | Prettyprinting | Interpreter (2) |
+| StableHLO Op (114) | Specification (21) | Verifier(104) | Type Inference(82) | Prettyprinting | Interpreter (2) |
```

Comment:
The current column titles are populated using GitHub label names. Let's stick to "Verification"?

Reply:
Done.